### ISSUE TITLE
RamFs entries returns directory entries as files

### ISSUE THREAD

**Issue by flaviojs** [germangb/mini-fs#6]
The version was bumped to 0.2 so I tried implementing a CaselessFs and found a problem with RamFs during my tests.

This code:
```rust
    let mut ram = RamFs::new();
    ram.touch("/a.txt", b"low a".to_vec());
    ram.touch("/A.TXT", b"high a".to_vec());
    ram.touch("/b/b.txt", b"low b".to_vec());
    ram.touch("/B/B.TXT", b"high b".to_vec());
    for entry in ram.entries("/").unwrap() {
        println!("{:?}", &entry);
    }
```
prints this:
```
Ok(Entry { name: "A.TXT", kind: File })
Ok(Entry { name: "a.txt", kind: File })
Ok(Entry { name: "B", kind: File })
Ok(Entry { name: "b", kind: File })
```
`B` and `b` are directories, not files.

**Comment by germangb** [comment]
Must be this (line 57): https://github.com/germangb/mini-fs/blob/e7887fe1de92b8d221f16b33225de861625d51a6/src/index.rs#L51-L63

the index.rs test should be updated to account for this

**Comment by germangb** [comment]
Should be fixed in `0.2.1`

**Comment by flaviojs** [comment]
Yup, it's fixed.
